src/main/js/field/input/SubmitButton.js

```diff
@@ -1,20 +1,18 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-const SubmitButton = ({ onClick, children, id }) => (
-    <button id={id} className="btn btn-md btn-primary" type="submit" onClick={onClick}>{children}</button>
+const SubmitButton = ({ children, id }) => (
+    <button id={id} className="btn btn-md btn-primary" type="submit">{children}</button>
```

Comment:
Everywhere this component is used is wrapped in a form element correct? Otherwise the submit won't work.

Reply:
yes, this is always part of a form. I added the onClick to master when working on this PR and that breaks a lot of things.